pages.id/common/abduco.md

```diff
@@ -0,0 +1,24 @@
+# abduco
+
+> Manajer sesi Terminal.
+> Informasi lebih lanjut: <http://www.brain-dump.org/projects/abduco/>.
```

Comment:
```suggestion
> Informasi lebih lanjut: <http://www.brain-dump.org/projects/abduco>.
```

Reply:
The English version clearly includes the trailing slash.

Reply:
> The English version clearly includes the trailing slash.

Yep, we shouldn't remove the trailing slash if the site redirects to it anyway, as a redirect will probably add up to load time a bit especially in slower connections.